src/stream/src/executor/wrapper/update_check.rs

```diff
@@ -31,20 +30,13 @@ pub async fn update_check(info: Arc<ExecutorInfo>, input: impl MessageStream) {
         let message = message?;
 
         if let Message::Chunk(chunk) = &message {
-            for ((op1, row1), (op2, row2)) in once(None)
-                .chain(chunk.rows().map(Some))
-                .chain(once(None))
-                .map(|r| (r.unzip()))
-                .tuple_windows()
-            {
-                if (op1.is_none() && op2 == Some(Op::UpdateInsert)) // the first row is U+
-                    || (op1 == Some(Op::UpdateDelete) && op2 != Some(Op::UpdateInsert))
-                {
-                    panic!(
-                        "update check failed on `{}`: expect U+ after  U-:\n first row: {:?}\nsecond row: {:?}",
-                        info.identity,
-                        row1,
-                        row2,
+            for record in chunk.records() {
```

Comment:
Oh, didn't find that there's also checking in `records()`. Would you please also refine the assertion message here?

https://github.com/risingwavelabs/risingwave/blob/921eefc1032250b24b3dfa5ae709bf87cf047633/src/common/src/array/stream_chunk_iter.rs#L90